docs/src/basics/Logging.md

```diff
@@ -0,0 +1,63 @@
+# Logging the Solve Process
+
+All NonlinearSolve.jl native solvers allow storing and displaying the trace of the nonlinear
```

Comment:
We definitely need to update https://docs.sciml.ai/NonlinearSolve/stable/basics/solve/#solver_options as well.